config/environments/development.rb

```diff
@@ -44,6 +44,8 @@
   # whitelist attributes for mass-assignment
   # config.active_record.whitelist_attributes = true
 
+  config.eager_load = false
+  
   # Set time zone
   config.time_zone = "Singapore"
```

Comment:
Do you want to mention this variable in your documentation.

Reply:
By default, this option is set to false for development environment. Here, I just decided to explicitly state it out.